src/views/DocumentationTopic.vue

```diff
@@ -89,6 +89,7 @@
           :languagePaths="languagePaths"
           :enableOnThisPageNav="enableOnThisPageNav"
           :enableMinimized="enableMinimized"
+          :disableHeroBackground="enableMinimized"
```

Comment:
To avoid having to make any changes in this file, I think you could instead update the `src/components/DocumentationTopic.vue` component to consider the already passed `enableMinimized` flag in the `enhanceBackground` computed property.

Also, it avoids an interesting scenario where the background could be enabled even when the minimal flag is passed in another context.

Reply:
The reason why I add this is because we want the background to always be disabled when minimal flag is on. And Dobri added the `diableHeroBackground` prop (that's not currently being used) to allow convenient disabling. I thought just using that was cleaner than making the `enhanceBackground` property even more complicated.

Reply:
That makes sense to me but since the `src/components/DocumentationTopic.vue` component can take both boolean flags and it doesn't use both in the logic, it's possible to have a situation where your first sentence is true when you use this view (I can show you a quick in-person demo of where I noticed this on something else I'm working on)